src/experiments/tests/gpid-prebid.ts

```diff
@@ -0,0 +1,29 @@
+import type { ABTest } from '@guardian/ab-core';
+
+export const gpidPrebidAdUnits: ABTest = {
+	id: 'GpidPrebidAdUnits',
+	author: '@commercial-dev',
+	start: '2024-11-15',
+	expiry: '2024-11-22',
```

Comment:
This expires in 2 days... maybe worth extending the date?

Reply:
👍 Ill extend for a month to match the frontend switch